cookiecutter/main.py

```diff
@@ -52,7 +52,8 @@ def expand_abbreviations(template, config_dict):
     return template
 
 
-def cookiecutter(template, checkout=None, no_input=False, extra_context=None):
+def cookiecutter(template, checkout=None, no_input=False,
```

Comment:
I would most certainly go for kwargs as the default value for ``extra_context`` *could* change in the future (I have doubts it wil :wink:).